experimental/Schemes/IdealSheaves.jl

```diff
@@ -512,6 +511,11 @@ function isone(I::IdealSheaf)
   return all(x->isone(I(x)), affine_charts(scheme(I)))
 end
 
+function isone(I::IdealSheaf, C::Covering)
```

Comment:
```suggestion
function isone(I::IdealSheaf; covering::Covering=default_covering(scheme(I)))
```
I would make this an optional case of the usual method. With the entailing changes below, i.e. `C->covering`.